### PR TITLE
chore: replace placeholder assets

### PR DESCRIPTION
**Pull request by flx92**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,11 +35,8 @@
     "deploy": "bash ./scripts/deploy.sh",
     "deploy:staging": "bash ./scripts/deploy-staging.sh",
     "serve": "browser-sync start --config bs-config.js --serveStatic docs --files docs",
-    "taint": "bash ./scripts/prepare-static.sh && rm -rf static && mkdir static && npm run taint:fonts && npm run taint:icons && npm run taint:images && npm run taint:videos",
+    "taint": "bash ./scripts/prepare-static.sh && rm -rf static/fonts && mkdir static/fonts && npm run taint:fonts",
```
Comment (marionebl) [inline package.json]: To be consistent we should remove the taint command completely. Please host the fonts from CDN

File: static/images/logos/lg_allianz.svg [hunk 2]
```diff
@@ -1 +1,31 @@
-<svg viewBox="0 0 134 60" height="60" width="134" xmlns="http://www.w3.org/2000/svg"><circle cx="67" cy="30" r="30" fill="#ffffff"/></svg>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (marionebl) [inline static/images/logos/lg_allianz.svg]: clean up via svgo

File: static/images/logos/lg_bmw.svg [hunk 3]
```diff
@@ -1 +1,29 @@
-<svg viewBox="0 0 134 60" height="60" width="134" xmlns="http://www.w3.org/2000/svg"><circle cx="67" cy="30" r="30" fill="#ffffff"/></svg>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (marionebl) [inline static/images/logos/lg_bmw.svg]: clean up via svgo

File: static/images/logos/lg_deutschebank.svg [hunk 4]
```diff
@@ -1 +1,11 @@
-<svg viewBox="0 0 134 60" height="60" width="134" xmlns="http://www.w3.org/2000/svg"><circle cx="67" cy="30" r="30" fill="#ffffff"/></svg>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (marionebl) [inline static/images/logos/lg_deutschebank.svg]: clean up via svgo

File: static/images/logos/lg_mcdonalds.svg [hunk 5]
```diff
@@ -1 +1,12 @@
-<svg viewBox="0 0 134 60" height="60" width="134" xmlns="http://www.w3.org/2000/svg"><circle cx="67" cy="30" r="30" fill="#ffffff"/></svg>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (marionebl) [inline static/images/logos/lg_mcdonalds.svg]: clean up via svgo

File: static/images/logos/lg_sinnerschrader.svg [hunk 6]
```diff
@@ -1 +1,5 @@
-<svg viewBox="0 0 134 60" height="60" width="134" xmlns="http://www.w3.org/2000/svg"><circle cx="67" cy="30" r="30" fill="#ffffff"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 133.68 18">
```
Comment (marionebl) [inline static/images/logos/lg_sinnerschrader.svg]: clean up via svgo

File: static/images/logos/lg_telefonica.svg [hunk 7]
```diff
@@ -1 +1,78 @@
-<svg viewBox="0 0 134 60" height="60" width="134" xmlns="http://www.w3.org/2000/svg"><circle cx="67" cy="30" r="30" fill="#ffffff"/></svg>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (marionebl) [inline static/images/logos/lg_telefonica.svg]: clean up via svgo

File: static/images/logos/lg_unitymedia.svg [hunk 8]
```diff
@@ -1 +1,253 @@
-<svg viewBox="0 0 134 60" height="60" width="134" xmlns="http://www.w3.org/2000/svg"><circle cx="67" cy="30" r="30" fill="#ffffff"/></svg>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (marionebl) [inline static/images/logos/lg_unitymedia.svg]: clean up via svgo

File: static/images/logos/lg_vw.svg [hunk 9]
```diff
@@ -1 +1,20 @@
-<svg viewBox="0 0 134 60" height="60" width="134" xmlns="http://www.w3.org/2000/svg"><circle cx="67" cy="30" r="30" fill="#ffffff"/></svg>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (marionebl) [inline static/images/logos/lg_vw.svg]: clean up via svgo

File: package.json (hunk 1 above)
Comment (flx92) [inline package.json]: You're right. But first of all we need a CDN.

**Comment by flx92** [comment]
added content images in PR #247 and refactored taint command in #251 and closed this PR.

**Pull request closed without merging**